concrete/src/Cache/Adapter/ZendCacheDriver.php

```diff
@@ -87,12 +138,14 @@ protected function internalGetItem(&$normalizedKey, &$success = null, &$casToken
      */
     protected function internalSetItem(&$normalizedKey, &$value)
```

Comment:
I don't know why, but yes, it's because the base ZF abstract class defines the abstract method this way.